### PR TITLE
CB-13353 : added save-exact option

### PR DESCRIPTION
**Pull request by audreyso**
### What does this PR do?
Added save-exact option

### Checklist
- [X] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [X] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [X] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -443,12 +444,16 @@ function cli (inputArgs) {
             });
         }
 
-        if (args.nosave) {
+        if (args.nosave || args['save-exact']) {
```
Comment (stevengill) [inline src/cli.js]: this change is probably not needed since we handle the save or save-exact or no-save flags in cordova-fetch.

Comment (audreyso) [reply src/cli.js]: kk updated! thanks!

File: src/cli.js [hunk 2]
```diff
@@ -449,6 +450,10 @@ function cli (inputArgs) {
             args.save = true;
         }
 
+        if (args['save-exact']) {
```
Comment (stevengill) [inline src/cli.js]: what happens if you don't have lines 453-456? Are they needed?

Comment (audreyso) [reply src/cli.js]: @stevengill Hmm you are right, I don't think these are needed. I will take it out. The test still passes without these lines and manual tests work too.

**Pull request closed without merging**